ddtrace/internal/runtime/tag_collectors.py

```diff
@@ -97,3 +98,21 @@ def collect_fn(self, keys):
         tags = super(PlatformTagCollectorV2, self).collect_fn(keys)
         tags.append(("runtime-id", get_runtime_id()))
         return tags
+
+
+class ProcessTagCollector(RuntimeTagCollector):
+    """Tag collector for process tags."""
+
+    def collect_fn(self, keys):
+        # DEV: we do not access direct process_tags_list so we can
+        # reload it in the tests
+        process_tags_list = process_tags.process_tags_list
+        if process_tags_list is None:
+            return []
+
+        tags = []
+        for tag_string in process_tags_list:
```

Comment:
Process tags should be generated once. Can we avoid parsing the values multiple times. We should be able to set these tags once

Reply:
Can you check that my new change is what you expected please ?